### ISSUE TITLE
FNE005 for built-in dunder methods

### ISSUE THREAD

**Issue by aklajnert** [Melevir/flake8-functions-names#16]
Small code example:
```python
def __eq__(self, other: Any) -> bool:
    return str(self) == str(other)
```

The plugin reports `FNE005 Return type of the function is bool, but the name doesn't show it`.  
Since these are built-in methods, there's no way to rename them. I think in such cases this error shouldn't be reported. Applies also to other methods like `__lt__()`, `__contains__()` etc.
